demo/llama/README.md

```diff
@@ -0,0 +1,65 @@
+The code in this directory trains the Llama 7B model on [Huggingface's RedPajama dataset](https://huggingface.co/datasets/togethercomputer/RedPajama-Data-1T-Sample/tree/main). All the code in this directory is borrowed from [Lightning AI's lit-llama repo](https://github.com/Lightning-AI/lit-llama). Changes have been made in appropriate places to fetch the data from GCS instead of reading from disk.
+
+This demo has been tested on a GCE instance with `2` Nvidia `H100` GPUs.
+
```

Comment:
Let's scope the demo to just fsdp checkpointing, if possible.

Reply:
Made changes to support saving checkpoints with Dataflux. Updated README with details. Could you take another look?

Reply:
The readme doesn't appear to cover checkpointing at all. Can you update it?

I think the training portion is not that interesting from readme purposes, since it doesn't involve Dataflux at all (does it?)

Reply:
I added a note in the "Run the pre-training script" section about implementing a custom strategy that makes it possible to save checkpoints using Dataflux. What other details would be good to mention here?